Task5/src/app.ts

```diff
@@ -28,11 +30,15 @@ export class App {
                 req.rawBody = buf;
             }
         }));
+        // this.app.use(cors());
```

Comment:
Uncomment this code and check if this is working.

Reply:
Added and working